ui/css/logviewer.css

```diff
@@ -47,12 +47,14 @@ body {
     background: #fff;
 }
 
-.lv-log-line {
-    padding: 0 18px;
+/* Equal weight selector needs to follow the above nth-child() */
+.lv-log-container > div.text-danger {
+    color: #db0000;
+    background: #fbe3e3;
 }
 
-.text-danger {
-    color: #db0000;
+.lv-log-line {
+    padding: 0 18px;
```

Comment:
I didn't actually add these two lines, just a peculiarity of the diff with the particular line changes.